### ISSUE TITLE
Cider "could not find artifact"

### ISSUE THREAD

**Issue by deg** [clojure-emacs/cider-nrepl#434]
## Expected behavior

Project expected to load under Cider

## Actual behavior

Project loads from command line, but not from cider-jack-in-clojurescript

Problem lies in finding dependency cljsjs/firebase. It gets loaded into ~/.m2, but not into project.
Key lines from log:

````
error in process sentinel: nrepl-server-sentinel: Could not start nREPL server: Retrieving cljsjs/firebase/3.2.1-0/firebase-3.2.1-0.pom from clojars
Retrieving cljsjs/firebase/3.2.1-0/firebase-3.2.1-0.jar from clojars
````
and
`Could not find artifact cljsjs:firebase:jar:3.2.1-0 in central (http://repo.maven.apache.org/maven2)`

## Steps to reproduce the problem

Full instructions at https://github.com/deg/trilystro/blob/bug-report-16Aug17/project.clj#L16-L60

TL;DR - Clone that project; remove band-aid line from project.clj; and try to load it.

Basically, it is a lein-based project, depending on a boot-based project, which depends on a cljsjs project. It load fine from the command line (`lein figwheel dev`) but not from Cider.

Adding the cljsjs dependency directly into the original project fixes the problem.

### cider-nrepl version

0.15.1-SNAPSHOT. Also verified in 0.15.0-SNAPSHOT

### Java version

````
$ /usr/bin/java -version
openjdk version "1.8.0_131"
OpenJDK Runtime Environment (build 1.8.0_131-8u131-b11-2ubuntu1.16.04.3-b11)
OpenJDK 64-Bit Server VM (build 25.131-b11, mixed mode)
````

### Operating system

Ubuntu 16.04 LTS

**Comment by bbatsov** [comment]
I think it's safe to assume this problem no longer exists.
